web/.env.example

```
@@ -37,6 +37,9 @@ CMUX_PRO_FROM_EMAIL=
 # checkout flow. Price ids are optional overrides; otherwise the app resolves
 # prices by lookup key in Stripe test/live mode.
 STRIPE_SECRET_KEY=
+# Set to 1 only after the Stripe account has tax registrations configured.
+# Leave at 0 (or unset) to keep Checkout tax calculation disabled.
+STRIPE_AUTOMATIC_TAX=0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_AUTOMATIC_TAX key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)